examples/layers/ios/.gitignore

```diff
@@ -0,0 +1,37 @@
+.idea/
```

Comment:
Looks like this covers a superset of #8062 - woohoo!

I'd actually started on a similar patch just covering the iOS side but stopped due to #9085, which would necessitate a re-do of that work. Not sure if you want to wait for a fix for that or